saleor/payment/gateways/braintree/errors.py

```diff
@@ -117,3 +127,23 @@ def get_error_type(error_code):
 
 class BraintreeException(Exception):
     pass
+
+
+BRAINTREE_ERROR_MESSAGES = {
+    AuthenticationError: "Authentication error",
+    AuthorizationError: "Authorization error",
+    NotFoundError: DEFAULT_ERROR_MESSAGE,
+    RequestTimeoutError: "Request timeout error",
+    UpgradeRequiredError: "Upgrade required error",
+    TooManyRequestsError: "Too many requests error",
+    ServerError: "Server error",
+    ServiceUnavailableError: "Server unavailable error",
+    GatewayTimeoutError: "Gateway timeout error",
+}
+
+
+def handle_braintree_error(exc):
+    msg = BRAINTREE_ERROR_MESSAGES.get(
+        type(exc), "Unexpected http response " + str(exc)
```

Comment:
`Unexpected http response` - is it correct? I would say that we had unexcpected paymet/braintree error, not sure if we have enough context here to say that the problem was related with http 🤔

Reply:
Got it from braintree - https://github.com/braintree/braintree_python/blob/master/braintree/util/http.py#L56

Reply:
Yeah, but if Braintree will add a new exception there, then we will raise an exception `Unexpected http response` as we will not have this error in the map, right? So not sure if the `default` error, shouldn't be more generic